Lib/test/test_cprofile.py

```diff
@@ -30,6 +30,38 @@ def test_bad_counter_during_dealloc(self):
 
             self.assertEqual(cm.unraisable.exc_type, TypeError)
 
+    @unittest.skipUnless(support.check_sanitizer(address=True), "only used to fail with ASAN")
```

Comment:
Is this particularly slow?
I was wondering why not test it for all builds?

Reply:
Not really. I added this just because that was the place where it failed. I can remove this so we can test it in all environments for regression.